app/controllers/carl_people_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CarlPeopleController < ApplicationController
+  #----------------- SHOW A PERSON --------------------
+  def show
+    @person = Person.find(params[:id])
+    @tasks_count = @person.tasks.count
+    @tasks_wip = @person.tasks.where(date_complete: nil).count
+    @tasks_done = @tasks_count - @tasks_wip
+  rescue
+    redirect_to "/person_not_found"
+  end
+
+  #----------------- SHOW A PERSON'S TASKS --------------------
+  def tasks
+    @person = Person.find(params[:id])
+    @tasks = Task.all.where(person_id: params[:id])
+
+    @people = [{
+        id: @person.id,
+        name: @person.name,
+        count: @person.tasks.where(date_complete: nil).count
+      }]
```

Comment:
This also has the benefit of catching more typos. If you typo the key of a hash (eg `@people.first[:naaaaaame]`) you'll get a nil and the page will render without the data. If you typo the method name (eg `@people.naaaaaame`) you'll get a `NoMethodErrror` with a backtrace showing you where the problem is.